docs/references/nextjs/get-auth.mdx

```diff
@@ -11,6 +11,10 @@ The `getAuth()` helper retrieves the authentication state, allowing you to prote
   If you are using App Router, use the [`auth()` helper](/docs/references/nextjs/auth) instead.
 </Callout>
 
+## Parameters
+
+`getAuth()` accepts one paramater: `req`, which is the Next.js request object.
```

Comment:
Doesn't it also take `opts`, or is this a different `getAuth`? Found it [here](https://github.com/clerk/javascript/blob/main/packages/nextjs/src/server/createGetAuth.ts#L19) in the JS respo

Reply:
damn good find !!!